portal/pom.xml

```diff
@@ -59,6 +59,10 @@
 			<artifactId>com.liferay.util.taglib</artifactId>
 		</dependency>
 		<dependency>
+			<groupId>com.liferay</groupId>
+			<artifactId>com.liferay.journal.api</artifactId>
```

Comment:
@juangon,
Shouldn't this dependency be in the Portal Showcase, not the Portal components?